### PR TITLE
fix(result): fix the result count show issue AP-768

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

Show correct counts info when search messages

**Which issue(s) this PR fixes** 🔨

AP-768

### CODE REVIEW

File: components/interactables/Search/Result/Result.vue [hunk 1]
```diff
@@ -143,6 +142,18 @@ export default Vue.extend({
         }
       },
     },
+    result: {
+      handler(newResult) {
+        if (newResult && newResult.data) {
+          this.$data.groupList.map((item: any) => {
```
Comment (molimauro) [inline components/interactables/Search/Result/Result.vue:148]: Please dont use `.map` if not returning anything. Just go for `for/forEach` much cleaner and efficient. Thanks

File: components/interactables/Search/Result/Result.vue [hunk 2]
```diff
@@ -143,6 +142,18 @@ export default Vue.extend({
...
+            if (item.title === 'Messages') {
```
Comment (molimauro) [inline components/interactables/Search/Result/Result.vue:149]: Will this `if` work if we change language?

File: components/interactables/Search/Result/Result.vue (hunk 1 above)
Comment (davidchan1219) [reply components/interactables/Search/Result/Result.vue:148]: Yes. I know that. That loop has return value. So I used map.

File: components/interactables/Search/Result/Result.vue (hunk 2 above)
Comment (davidchan1219) [reply components/interactables/Search/Result/Result.vue:149]: I changed the condition as "type".

**Comment by phillsatellite** [comment]
Tested: Seems like whenever user tries to use search bar they get error: "_item$payload.toLowerCase is not a function. Tested Locally when I got this crash

Uploading crash.mov…


https://user-images.githubusercontent.com/93608357/153944962-2dccdb2c-1ef6-4e39-a572-1189f1a7ab03.mov

**Comment by davidchan1219** [comment]
@phillsatellite I just fixed the code. Could you test the above case with new code again?

**Comment by WanderingHogan** [comment]
Closing, we will try to incorporate these in the search branch

**Pull request closed without merging**